samples/hello-kapt/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+import org.gradle.api.internal.HasConvention
```

Comment:
:x: unused import

Reply:
Yeah. Also strange, that Idea doesn't show this (and some other) import as unused